lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1753,11 +1785,9 @@ class HybridTextEditing {
   ///
   /// The constructor also decides which text editing strategy to use depending
   /// on the operating system and browser engine.
-  HybridTextEditing() {
-    channel = TextEditingChannel(this);
-  }
+  HybridTextEditing();
 
-  late TextEditingChannel channel;
+  late final TextEditingChannel channel = TextEditingChannel(this);
```

Comment:
Does this still need `late`?

I feel like I'm wrong and I got tripped up by this recently...

Reply:
I think `late` is still required because the right-hand-side expression refers to `this`.

Reply:
Yes. This needs lazy initialization because `this` has to be initialized first, to be passed to the `TextEditingChannel` constructor, otherwise we'll have a circular dependency.